org.lflang/src/org/lflang/generator/GeneratorBase.xtend

```diff
@@ -1113,7 +1113,7 @@ abstract class GeneratorBase extends AbstractLFValidator {
         // In case errors occur within an imported file, record the original path.
         val originalPath = path;
         
-        var severity = IMarker.SEVERITY_ERROR
+        var severity = IMarker.SEVERITY_WARNING
```

Comment:
What is the reason for this change?

Reply:
To treat any output from the compiler as a warning until we find an actual instance of an error. Sometimes a warning seem to have been interpreted as an error.

Reply:
Got it. Thanks!